README.rst

```diff
@@ -93,47 +103,48 @@ Dependencies
 
 The minimum required dependencies to run MNE-Python are:
 
-- Python >= 3.8
-- NumPy >= 1.21.2
-- SciPy >= 1.7.1
-- Matplotlib >= 3.5.0
-- pooch >= 1.5
+- Python ≥ 3.8
+- NumPy ≥ 1.21.2
+- SciPy ≥ 1.7.1
+- Matplotlib ≥ 3.5.0
+- Pooch ≥ 1.5
```

Comment:
does `≥` work when copy pasting and pip installing? If not, I'd prefer the old syntax.

Reply:
It doesn't, but I don't think this list is intended to be copied and pasted.

Reply:
No strong opinion from me here. @drammock ?

Reply:
they weren't copy-pastable before either because of the spaces around the operators. I agree with @cbrnr that the intent of this list is to be read, not copy-pasted.